src/Command/Branch/BranchChangelogCommand.php

```diff
@@ -87,7 +138,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $commits = $this->getHelper('git')->getLogBetweenCommits($latestTag, $branch);
         $issues = $this->getIssuesFromCommits($commits, $input->getOption('search'));
 
-        foreach ($issues as $id => $idLabel) {
+        $output->writeln(sprintf('<info>Found %s commits</info>', count($commits)), OutputInterface::VERBOSITY_DEBUG);
+
+        $items = [];
+
```

Comment:
I try to keep language constructs and variable definitions separate for better readability 👍